cypress/integration/core-opensearch-dashboards/opensearch-dashboards/dashboard_sample_data_with_datasource_spec.js

```diff
@@ -45,6 +45,31 @@ if (Cypress.env('DATASOURCE_MANAGEMENT_ENABLED')) {
       });
     });
 
+    describe('checking legacy home page', () => {
+      before(() => {
+        // Go to the home page
+        miscUtils.visitPage('app/home#');
```

Comment:
have we checked with the security plugin installed?

Reply:
While I fixed a typo here, this piece is an existing code. If it was being checked before, it will be checked now too.